.vscode/launch.json

```diff
@@ -4,6 +4,25 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+    {
+      "name": "API",
+      "type": "python",
+      "request": "launch",
+      "module": "uvicorn",
+      "args": ["api:app", "--reload", "--port", "5002"],
+      "jinja": true,
```

Comment:
Do we need jinja for an API if we have FLask?

Reply:
you r right - no need